server/inngest/eventHandlers/sendEventReminders.ts

```
@@ -0,0 +1,203 @@
+      );
+
+      // Process each subscription
+      for (const subscription of subscriptions) {
+        try {
+          const hemocioneId = subscription.hemocioneId;
+
+          // Get user's notification preferences
+          const preferences = await getNotificationPreference(hemocioneId);
+
+          // Default channels to try if no preferences exist
+          const channelsToTry: NotificationChannel[] = ["sms", "whatsapp", "email"];
```

ESLint

[error] Replace `"sms",·"whatsapp",·"email"` with `⏎············"sms",⏎············"whatsapp",⏎············"email",⏎··········`

(prettier/prettier)